src/lib/php/common-ui.php

```
@@ -311,11 +311,34 @@ function Get1stUploadtreeID($upload)
  */
 function Convert2BrowserTime($server_time)
 {
-  $server_timezone = date_default_timezone_get();
-  $browser_time = new \DateTime($server_time, new \DateTimeZone($server_timezone));
-  if (array_key_exists("timezone", $_SESSION)) {
-    $tz = $_SESSION["timezone"];
-    $browser_time->setTimeZone(new \DateTimeZone($tz));
+  try {
+    if (empty($server_time)) {
+      throw new \InvalidArgumentException('Server time cannot be empty');
+    }
+    $server_timezone = date_default_timezone_get();
+
+    try {
+      $browser_time = new \DateTime($server_time, new \DateTimeZone($server_timezone));
+    } catch (\Exception $e) {
+      error_log("Invalid server time format: $server_time");
+      return false;
```

If a failure occurs, throw an exception instead of returning `false`. This ensures consistent return values and shifts error handling to the caller.

Or Better :
If there is an exception `return $server_time` with the format: `format('Y-m-d H:i:s')`

This also satisfies the goal of, browser time into server timezone.

---

removed all false returns. now it wil return formatted date('Y-m-d H:i:s'), ensuring consistent return values.